### DISCUSSION TITLE
pagination total 传入数字 显示 {total}

### DISCUSSION THREAD

**Discussion by 18378559745** [element-plus/element-plus#4739] [Q&A] [answered]
<el-pagination
            :page-sizes="[10, 20, 30, 40,50]"
            :current-page="page"
            :page-size="page_size"
            :total="100"
            layout=" total, sizes, prev, pager, next, jumper"
            @size-change="handleSizeChange"
            @current-change="handleCurrentChange"
            style="text-align: center;margin-top:20px"
        >
        </el-pagination>

**Comment by sxzz** [comment, marked as answer]
已修复，等待新版本发布
